src/constructor.ts

```diff
@@ -118,10 +111,31 @@ function makeDomainFunction<
               : formatSchemaErrors(envResult.error.issues),
           })
         }
-        return handler(result.data, envResult.data)
+        return handler(result.data as I, envResult.data as E)
       })
     } as DomainFunction<Awaited<Output>>
   }
 }
 
+// deno-lint-ignore ban-types
+const objectSchema: ParserSchema<{}> = {
+  // deno-lint-ignore require-await
```

Comment:
To remove this comment you can remove the `async` and return `Promise.resolve({ success: true, data })`